src/tests/e2e/pages/emailTransferAuth.page.js

```diff
@@ -1,9 +1,18 @@
+import MetamaskPage from './metamask.page'
 import log from 'loglevel'
 log.setDefaultLevel('info')
 
 class EmailTransferAuthPage {
-  async connect () {
+  constructor () {
+    this.metamaskPage = new MetamaskPage()
```

Comment:
I think this should be in the connect method instead of the constructor.

Reply:
fixed

Reply:
fixed